IPR Policy.md

```diff
@@ -84,11 +84,19 @@ If the [Living Standard] is depublished by the WHATWG, then no new licenses need
 
 "Day" means 24 hours, commencing upon execution of a specific act, and is calculated without reference to time zones. To avoid confusion, Universal Coordinated Time is used.
 
+### 2.10. "Field of Web Technologies"
+
+"Field of Web Technologies" means the field of creating and influencing web standards and web technologies that could be adopted by the web community. "Field of web technologies" does not include implementation of web standards solely for the purpose of creating a website or app.
+
+### 2.10.1. "Work in the Field of Web Technologies"
+
+"Work in the Field of Web Technologies" means that the contributor's role at the contributor's company or entity is in the [field of web technologies]. If the contributor's role within its company or entity does not involve the field of web technologies, as defined here, the contributor does not "work in the field of web technologies" regardless of whether the company's or entity's work is generally within the field of web technologies.
```

Comment:
This sounds like a change where I can finally sign the participation agreement as an individual rather than an employee of my company. However, if in the future I get reassigned to a role that does influence web standards and web tech, what should happen?

Reply:
See e.g., https://github.com/whatwg/participant-data/commits?author=TimothyGu, when I was intermittently hired and not hired by a company that is in the field of Web technologies due to internships.